app/controllers/kaui/subscriptions_controller.rb

```diff
@@ -118,13 +118,14 @@ def destroy
       # START_OF_TERM is *not* a valid entitlement_policy and so would default to IMMEDIATE
       entitlement_policy = billing_policy && billing_policy == 'START_OF_TERM' ? 'IMMEDIATE' : billing_policy
 
-      # true by default
-      use_requested_date_for_billing = (params[:use_requested_date_for_billing] || '1') == '1'
-
+      # true by default except default policy
+      use_requested_date_for_billing = if requested_date || billing_policy
```

Comment:
I just re-read the backend code: `use_requested_date_for_billing` is only used when a `requested_date` is used. You could simplify here the logic to simply `if requested_date`.